zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -291,4 +302,31 @@ public static Either<ProblemDetail, UserResponse> toUser(final UserEntity user)
             .email(user.email())
             .name(user.name()));
   }
+
+  private static List<DecisionInstanceItem> toDecisionInstances(
+      final List<DecisionInstanceEntity> instances) {
+    return instances.stream().map(SearchQueryResponseMapper::toDecisionInstance).toList();
+  }
+
+  private static DecisionInstanceItem toDecisionInstance(final DecisionInstanceEntity entity) {
+    return new DecisionInstanceItem()
+        .key(entity.key())
+        .state(
+            (entity.state() == null)
+                ? null
+                : DecisionInstanceStateEnum.fromValue(entity.state().name()))
+        .evaluationDate(entity.evaluationDate())
+        .evaluationFailure(entity.evaluationFailure())
+        .processDefinitionKey(entity.processDefinitionKey())
+        .processInstanceKey(entity.processInstanceKey())
+        .decisionKey(Long.valueOf(entity.decisionId()))
```

Comment:
`decisionId` is a string like `invoiceClassification`. So I guess we need to check if we can even use `decisionKey` here.
![image](https://github.com/user-attachments/assets/ef51f3a4-1e82-4b6b-a7e2-4e1e464667f6)

Reply:
indeed, I swapped decisionId and decisionDefinitionId , it should be fixed now